cpu/esp32/Makefile.dep

```diff
@@ -16,7 +18,7 @@ ifneq (,$(filter esp_eth,$(USEMODULE)))
   USEMODULE += esp_idf_eth_phy
   USEMODULE += netdev_eth
   USEMODULE += netopt
-  USEMODULE += xtimer
+  USEMODULE += ztimer_msec
```

Comment:
If `ztimer_usec` is used anyway, would it be better leave everything using `ztimer_usec`?

Reply:
True